### PR TITLE
Ch 219

### PR DESCRIPTION
**Pull request by ryuheimat**
Fix issues of 
[CH-219]

### CODE REVIEW

File: redux/actions/template/index.ts [hunk 1]
```diff
@@ -247,53 +239,53 @@ const getContractTemplate = (
...
+                    //   type: 'string',
+                    // },
+                    // anOption: {
+                    //   title: 'An option',
+                    //   type: 'string',
+                    // },
+                    // exercise: {
+                    //   title: 'Exercise',
+                    //   type: 'string',
+                    // },
```
Comment (CoolHandLuke88) [inline redux/actions/template/index.ts]: remove dead code

File: redux/actions/template/index.ts [hunk 2]
```diff
@@ -247,53 +239,53 @@ const getContractTemplate = (
...
+                    // aRight: {
+                    //   title: 'A right',
+                    //   type: 'string',
+                    // },
+                    // purchase: {
+                    //   title: 'Purchase',
+                    //   type: 'string',
+                    // },
```
Comment (CoolHandLuke88) [inline redux/actions/template/index.ts]: remove dead code

File: redux/actions/template/index.ts [hunk 3]
```diff
@@ -79,14 +79,8 @@ const getContractTemplate = (
...
-    wallet: {
-    },
-    required: [
-      'partyName',
-      'partyAddress',
-      'partyEmail',
-      'partyWallet',
-    ],
+    wallet: {},
+    required: ['partyName', 'partyAddress', 'partyEmail', 'partyWallet'],
```
Comment (CoolHandLuke88) [inline redux/actions/template/index.ts]: disagreement on linting/styling?

File: redux/actions/template/index.ts [hunk 4]
```diff
@@ -28,11 +28,11 @@ const getContractTemplate = (
...
-  let contractTemplate
-  let title
-  let dataName = ''
-  let jsonSchemas: any = []
-  let uiSchema: Object = {}
+  let contractTemplate;
+  let title;
+  let dataName = '';
+  let jsonSchemas: any = [];
+  let uiSchema: Object = {};
```
Comment (CoolHandLuke88) [inline redux/actions/template/index.ts:35]: do we not have a shared config for linting?

File: redux/actions/template/index.ts (hunk 1 above)
Comment (ryuheimat) [reply redux/actions/template/index.ts]: it's already removed

File: redux/actions/template/index.ts [hunk 5]
```diff
@@ -1014,8 +1015,10 @@ const getContractTemplate = (
             title: 'State name',
             type: 'string',
           },
-          required: ['stateOfCompany'],
-        }, */
+
+        },
+        required: ['stateOfCompany'],
+      }, */
```
Comment (CoolHandLuke88) [inline redux/actions/template/index.ts:981]: remove dead code

File: redux/actions/template/index.ts (hunk 4 above)
Comment (ryuheimat) [reply redux/actions/template/index.ts:35]: we have .eslintrc

File: redux/actions/template/index.ts (hunk 5 above)
Comment (ryuheimat) [reply redux/actions/template/index.ts:981]: it's not my code

**Review by CoolHandLuke88 — APPROVED** [review]

**Pull request merged**